### DISCUSSION TITLE
Hi, I am using run_pycistarget for zebrafish analysis, I am wondering which species name should I use? I tried species = 'danio_rerio'; or species = 'drerio', but none of the names work.

### DISCUSSION THREAD

**Discussion by ljljolinq1010** [aertslab/pycistarget#12] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @ljljolinq1010 

I transferred your issue to the discussions because it's more a question and not a bug.

When you say it does not work I guess you're getting the following error message?

```python
Unable to load annotation for danio_rerio
```

The reason for this is that we don't provide motif-to-TF annotations for zebrafish at this moment (we only have human, fly and mouse).  You can generate your own custom annotation based on our annotations (for example based on orthology). 

Once you've done this you can provide this custom annotation using the parameter `path_to_motif_annotations`.

I hope this clarifies things.

Best,

Seppe

**Reply by ljljolinq1010** [reply]
I see, thanks a lot for your quick and informative reply. very cool method.

**Reply by mtrebelo** [reply]
Hi @ @ljljolinq1010 

I am also running SCENIC+ with zebrafish data (danRer11). Which data did you use for the **rankings_db**, **scores_db** and **motif_annotation** at the 'Cistarget databases' point of the tutorial?

Thanks a lot in advance.
